tests/test_validation.py

```diff
@@ -208,6 +208,53 @@ def test_conflict_in_dependencies():
         obj.validate(v1, 'first')
 
 
+def test_array_no_items_pass():
+    """Test that having an array without items passes."""
+    v1 = {
+        "$schema": "http://json-schema.org/draft-04/schema#",
+        "type": "object",
+        "properties": {
+            "$schema": {"type": "string"},
+            "_files": {
+                "type": "array"
+            }
+        }
+    }
+    obj = JSONSchemaValidator()
+    obj.validate(v1, 'first')
+
+
+def test_enum_in_array_items_pass():
+    """Test that having enum in array's items passes."""
+    v1 = {
+        "$schema": "http://json-schema.org/draft-04/schema#",
+        "type": "object",
+        "properties": {
+            "$schema": {"type": "string"},
+            "resource_type": {
+                "title": "Resource Type",
+                "description": "The type of the resource.",
+                "type": "array",
+                "items": {
```

Comment:
Is this tested? The code above seems to test only the case where `items` is a list.